infrastructure/cloud/environments/dev/dev.tfvars

```diff
@@ -0,0 +1,5 @@
+app_name            = "jasper"
+environment         = "dev"
+kms_key_name        = "jasper-kms-key"
```

Comment:
Can we change reading  these variables from environment variables instead of files? Some of the values may be sensitive and we dont want storing them in files in repo. What we can do is, set environment variables or secrets on Github repo then read those here. Any terraform variables need to be in format TF_VAR_name